packages/db/postgres/migrations/009-migrate-backup-urls.sql

```diff
@@ -0,0 +1 @@
+UPDATE "upload" SET "backup_urls" = (SELECT array_agg("url") FROM "backup" WHERE "upload_id" = "upload"."id");
```

Comment:
Now that the schema is the same (`text[]` instead of `jsonb`), the [nft batch script](https://gist.github.com/yocontra/4ae719b42a54c4582ca28fcac74199d0) works without modifications. I tested it locally too.

Reply:
Yeah, I think batch script is best way to go. 

@hugomrdias how long did it take for you? We should probably schedule a maintenance windows

Reply:
awesome ok thanks, I will setup a migration window for this then.
@francois-potato we should be ready to merge here and do the maintenance next monday right?